src/tests/ftest/util/data_mover_utils.py

```diff
@@ -0,0 +1,167 @@
+        param_names.sort(key='dest_path'.__eq__)
+
+        return param_names
+
+    def set_datamover_params(self, src_pool=None, dst_pool=None, src_cont=None,
+                             dst_cont=None, display=True):
+        """Set the datamover params for the DAOS group, pool, and cont uuid.
+
+        Args:
+          src_pool(TestPool): source pool object
+          dst_pool(TestPool): destination pool object
+          src_cont(TestContainer): source container object
+          dst_cont(TestContainer): destination container object
+          display (bool, optional): print updated params. Defaults to True.
+        """
+
+        # set the obtained values
+        if src_pool:
+            self.daos_src_pool.update(src_pool.uuid,
+                                      "daos_src_pool" if display else None)
+            # setting src service level
+            src_svcl = ":".join(
+                [str(item) for item in [
+                    int(src_pool.pool.svc.rl_ranks[index])
+                    for index in range(src_pool.pool.svc.rl_nr)]])
+            self.daos_svcl.update(src_svcl, "svcl" if display else None)
+#            self.daos_src_svcl.update(src_svcl, "svcl" if display else None)
+
+        if dst_pool:
+            self.daos_dst_pool.update(dst_pool.uuid,
+                                      "daos_dst_pool" if display else None)
+            # setting destination service level
+#            dst_svcl = ":".join(
+#                [str(item) for item in [
+#                    int(dst_pool.pool.svc.rl_ranks[index])
+#                    for index in range(dst_pool.pool.svc.rl_nr)]])
+#            self.daos_dst_svcl.update(dst_svcl, "svcl" if display else None)
+
+        if src_cont:
+            self.daos_src_cont.update(src_cont.uuid,
+                                      "daos_src_cont" if display else None)
+        if dst_cont:
+            self.daos_dst_cont.update(dst_cont.uuid,
+                                      "daos_dst_cont" if display else None)
+
+
+class DataMover(DataMoverCommand):
+    """Class defining an object of type DataMoverCommand."""
+
+    def __init__(self, hosts, timeout=30):
+        """Create a datamover object."""
+        super(DataMover, self).__init__(
+            "/run/datamover/*",
+            "/home/standan/mpiio/install/mpifileutils/bin/dcp")
+
+        # set params
+        self.timeout = timeout
+        self.hosts = hosts
+
+    def run(self, tmp, processes):
+        """Run the datamover command.
+
+        Args:
+            check (bool): Check if datamover mounted properly after
+                mount is executed.
+        Raises:
+            CommandFailure: In case datamover run command fails
+
+        """
+        self.log.info('Starting datamover')
+
+        # Get job manager cmd
+        mpirun = Mpirun(self, mpitype="mpich")
+        mpirun.assign_hosts(self.hosts, tmp)
+        mpirun.assign_processes(processes)
+
+        # run dcp
+        out = mpirun.run()
```

Comment:
I don't mind having that in, but what do you think about just using it when you initialize your datamover/dcp object in your test?
dcp = DataMover(self.hostlist_clients)
.
.
dcp.exit_status_exception = False

Reply:
The problem is that I need the `mpirun` object created by `Mpirun(self, mpitype="mpich")` to set
`exit_status_exception = False`
And since the `mpirun` object is created within the scope of this function, I don't have access to it.
So even if I set
`dcp.exit_status_exception = False`
the `mpirun` object still has the default of `mpirun.exit_status_exception = True`

Reply:
I see what you mean. I have added that param in there.